app/src/main/java/com/topjohnwu/magisk/ui/flash/FlashFragment.kt

```diff
@@ -62,6 +63,10 @@ class FlashFragment : BaseUIFragment<FlashViewModel, FragmentFlashMd2Binding>()
         super.onDestroyView()
     }
 
+    override fun onKeyEvent(event: KeyEvent): Boolean {
+        return event.keyCode == KeyEvent.KEYCODE_VOLUME_UP || event.keyCode == KeyEvent.KEYCODE_VOLUME_DOWN
```

Comment:
This doesn't look really nice and additional keys will result in ugly commits.

This would be way nicer:
```kotlin
return when(event.keyCode) {
    KeyEvent.KEYCODE_VOLUME_UP,
    KeyEvent.KEYCODE_VOLUME_DOWN -> true
    else -> false
}
```